### PR TITLE
made some changes to the structure based on docs

### PR DESCRIPTION
**Pull request by rbrooks95**
@rmnegatives based on the docs its saying I should create a crud.py file [docs](https://fastapi.tiangolo.com/tutorial/sql-databases/#crud-utils) i guess this is best practices or i can just implement straight to interact with the database. let me know!

### CODE REVIEW

**Comment by rmnegatives** [comment]
> @rmnegatives based on the docs its saying I should create a crud.py file [docs](https://fastapi.tiangolo.com/tutorial/sql-databases/#crud-utils) i guess this is best practices or i can just implement straight to interact with the database. let me know!

Yea, this doc is EXCELLENT.. all of this is the level of "production" 
Follow through it all the way to the end before this year ends.

File: pagination/models.py [hunk 1]
```diff
@@ -15,12 +14,16 @@ class Comment(Base):
     timestamp = Column(DateTime(timezone=True), server_default=func.now())
     post_id = Column(Integer, ForeignKey("post.id"))
 
+    items = relationship("Post", back_populates="content")
+
 
 class Post(Base):
-    __tabelname__ = "post"
```
Comment (rmnegatives) [inline pagination/models.py:20]: you could have left this as post.
Usually it's implied that it contains "posts" as a db table would rarely hold one row of data.

**Comment by rmnegatives** [comment]
PR looks good.

**Comment by rbrooks95** [comment]
> PR looks good.

alright updated new code for the most part im finished. may just clean over time @rmnegatives

**Pull request merged**